lib/process-services/src/lib/task-list/components/task-form/task-form.component.html

```diff
@@ -59,15 +59,17 @@ <h4>
                     </adf-empty-content>
                 </ng-template>
             </mat-card-content>
-            <mat-card-actions class="adf-task-form-actions">
-                <ng-template [ngTemplateOutlet]="taskFormButtons"></ng-template>
-                <button mat-button
-                    *ngIf="!isCompletedTask()" id="adf-no-form-complete-button"
-                    color="primary"
-                    [disabled]="canCompleteNoFormTask()"
-                    (click)="onCompleteTask()">
-                    {{'ADF_TASK_FORM.EMPTY_FORM.BUTTONS.COMPLETE' | translate}}
-                </button>
+            <mat-card-actions>
+                <div class="adf-task-form-actions"> 
```

Comment:
Why did you introduced additional div here?

Reply:
In order to eliminate the necessity of relying on the internal material classes, we created a wrapper div and added styling in accordance with our specifications.

Reply:
I can't see any styles related to this class, is it used anywhere?

Reply:
In line number 6 of (lib/process-services/src/lib/task-list/components/task-form/task-form.component.scss)